lib/vsc/filesystem/operator.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: latin-1 -*-
+#
+# Copyright 2009-2022 Ghent University
+#
+# This file is part of vsc-filesystems,
+# originally created by the HPC team of Ghent University (http://ugent.be/hpc/en),
+# with support of Ghent University (http://ugent.be/hpc),
+# the Flemish Supercomputer Centre (VSC) (https://www.vscentrum.be),
+# the Flemish Research Foundation (FWO) (http://www.fwo.be/en)
+# and the Department of Economy, Science and Innovation (EWI) (http://www.ewi-vlaanderen.be/en).
+#
+# https://github.com/hpcugent/vsc-filesystems
+#
+# All rights reserved.
+#
+"""
+Interface to dynamically load vsc.filesystem modules and instantiate its XxxxOperations class
+
+@author: Alex Domingo (Vrije Universiteit Brussel)
+"""
+import logging
+
+def load_storage_operator(storage):
+    """
+    Load and initialize corresponding operator class for this filesystem
+    Return *Operations object
+
+    @type storage: Storage object from vsc.config
+    """
+    if getattr(storage, 'backend_operator', None):
+        return storage.backend_operator
+
+    Operator, OperatorError = import_operator(storage.backend)
+
+    try:
+        storage.backend_operator = Operator(**storage.api)
```

Comment:
@kwaegema can you paste LsutreOperator init code you use now?

Reply:
I now  do it like this, because i didnt want to touch the operations init: 
```
        if self.storage.fsbackend == 'lustre':
            self.fsops = LustreOperations()
            self.fsops.set_default_mapping(self.storage.mapping)
        elif self.storage.fsbackend == 'gpfs':
            self.fsops = GpfsOperations()
```

Reply:
with the `custom` attribute, I could change the lustre code to do the mapping in the init based on info from custom dict/file

Reply:
I added the lustre backend and all backend now get the`operator_config` dict from `Storage` passed into their init. See https://github.com/hpcugent/vsc-filesystems/pull/83/commits/45c783129760e596848eba5f537d64930beb32cc